internal/api/impl/v1/dashboard/schemas/hotreload.go

```diff
@@ -0,0 +1,132 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package schemas
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/perses/common/async"
+	"github.com/sirupsen/logrus"
+	log "github.com/sirupsen/logrus"
+	"gopkg.in/fsnotify.v1"
+)
+
+type watcher struct {
+	async.Task
+	fsWatcher   *fsnotify.Watcher
+	schemasPath string
+	validator   Validator
+}
+
+type reloader struct {
+	async.Task
```

Comment:
```suggestion
async.SimpleTask
```
Since now you don't use the method `Initialize` and `Finalize` for the reloader